File: specification/specs/analysis/analyzers/FingerprintAnalyzer.ts
class FingerprintAnalyzer extends AnalyzerBase {
	@prop_serializer("NullableStringIntFormatter")
	max_output_size: integer;
	@prop_serializer("NullableStringBooleanFormatter")
	preserve_original: boolean;
	separator: string;
	stopwords: StopWords;
	stopwords_path: string;
}

File: output/csharp/Analysis/Analyzers/FingerprintAnalyzer.cs
using Nest.Internal;
using Nest.Analysis;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Analysis {

	public class FingerprintAnalyzer : AnalyzerBase {
		
		[DataMember(Name="max_output_size")]
		public int MaxOutputSize { get; set; }


		[DataMember(Name="preserve_original")]
		public bool PreserveOriginal { get; set; }


		[DataMember(Name="separator")]
		public string Separator { get; set; }


		[DataMember(Name="stopwords")]
		public StopWords Stopwords { get; set; }


		[DataMember(Name="stopwords_path")]
		public string StopwordsPath { get; set; }

	}
}
